resources/ddl/define-all.spark

```diff
@@ -530,6 +530,9 @@ sqlContext.sql("CREATE TEMPORARY FUNCTION lr_datagen AS 'hivemall.dataset.Logist
 sqlContext.sql("DROP TEMPORARY FUNCTION IF EXISTS f1score")
 sqlContext.sql("CREATE TEMPORARY FUNCTION f1score AS 'hivemall.evaluation.FMeasureUDAF'")
```

Comment:
It is better to copy the old FMeasureUDAF.java as F1ScoreUDAF.java for a backward compatibility.

Then, `CREATE TEMPORARY FUNCTION f1score AS 'hivemall.evaluation.F1ScoreUDAF'`.